### PR TITLE
make git quiet unless verbose logging is enabled

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/source-to-image/issues/766

### CODE REVIEW

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
@openshift/devex ptal

File: pkg/scm/git/git.go [hunk 1]
```diff
@@ -13,6 +13,8 @@ import (
 	"strconv"
 	"strings"
 
+	log "github.com/golang/glog"
```
Comment (coreydaley) [inline pkg/scm/git/git.go]: Why log instead of glog as used elsewhere?

Comment (bparees) [inline pkg/scm/git/git.go]: glog is a wrapper in this file.  if you want to know the actual loglevel that's been set (as I do in the new logic), you need to use the real glog package to check.

Comment (bparees) [inline pkg/scm/git/git.go]: and if you just mean why didn't i import it as "glog" it's because the wrapper variable is named glog.

**Comment by jim-minter** [comment]
lgtm

**Comment by bparees** [comment]
[merge][severity:lowrisk]

**Pull request merged**